### PR TITLE
Added "Health" property, changed "Potion" to "Thrown Potion"

### PR DESCRIPTION
**Pull request by T-727**
- Added "Health" property, changed "Potion" to "Thrown Potion"
- Changed "Potion" to "Thrown Potion"

### CODE REVIEW

**Comment by 11-90-an** [comment]
evoker fangs (898) and small fireball (966) are empty, Not Applicable for both maybe?

**Comment by T-727** [comment]
@11-90-an done
tyvm

File: data/entity_data.json [hunk 1]
```diff
@@ -864,6 +864,142 @@
 				},
 				"Fishing Bobber": 0.2125
 			}
+		},
+	"properties": {
+		"health": {
+			"property_name": "Health": ,
+			"default_value": "Undefined": ,
+			"default_selection": true,
```
Comment (JoakimThorsen) [inline data/entity_data.json:872]: You have a few extra `:` characters here

File: data/entity_data.json [hunk 2]
```diff
@@ -864,6 +864,142 @@
...
+			"entries": {
+		"Area Effect Cloud": "Not Applicable",
+		"Armor Stand": 20,
+		"Arrow": "Not Applicable",
+		"Axolotl": 14,
+		"Bat": 6,
+		"Bee": 10,
+		"Blaze": 20,
+		"Boat": ,
```
Comment (JoakimThorsen) [inline data/entity_data.json]: Missing Value

File: data/entity_data.json [hunk 3]
```diff
@@ -864,6 +864,142 @@
...
+			"Other": 20
+		},
+		"Parrot": 6,
+		"Phantom": 20,
+		"Pig": 10,
+		"Piglin": 16,
+		"Piglin Brute": 50,
+		"Pillager": 24,
+		"Polar Bear": 30,
+		"Primed TNT": ,
```
Comment (JoakimThorsen) [inline data/entity_data.json]: Missing Value

File: data/entity_data.json [hunk 4]
```diff
@@ -864,6 +864,142 @@
...
+		"Chicken": 4,
+		"Cod": 3,
+		"Cow": 10,
+		"Creeper": 20,
+		"Dolphin": 10,
+		"Donkey": "15-30",
+		"Dragon Fireball": "Not Applicable",
+		"Drowned": 20,
+		"Elder Guardian": 80,
+		"End Crystal": ,
```
Comment (JoakimThorsen) [inline data/entity_data.json]: Missing Value

**Review by JoakimThorsen — CHANGES_REQUESTED** [review]
There are still a few missing values and you have a few JSON formatting errors, including one too few closing curly braces. I'd recommend verifying your formatting by using something like https://jsonlint.com/

**Comment by JoakimThorsen** [comment]
That's better :P Thanks

**Pull request merged**